tools/gen_parser/skulpt_parser_genarator.py

```diff
@@ -73,54 +73,20 @@
 """
 
 # todo mangle names like await
-reserved = {"await"}
+reserved = {"await", "arguments"}
 
 
 def fix_reserved(name):
     return name + "_" if name in reserved else name
 
 
-import re
-
-
 def clean_type(t):
-    t = t.replace("_ty", "").replace("*", "").replace("asdl_seq", "any")
-    if t == "arguments":
-        return "arguments_"
-    elif t == "AugOperator":
-        return "operator"
     return t
 
 
 def clean_action(action):
-    action = re.compile(r" -> v . [\w]+ . ([\w+])").sub(r".\1", action)
-
-    action = re.compile(r"\( [\w_]+ \* \)").sub("", action)  # type check
-    # action = re.compile(r" ([A-Z]+[a-z]+[\w]*) ").sub(r" new astnodes.\1 ", action)  # astnode constructor
-    action = re.compile(r"asdl_\w+ \*? ,").sub("", action)  # type check
-    # action = re.compile(r", p -> arena |, p | p ,").sub("", action)
-
-    action = (
-        # action.replace("_Py_", "new astnodes.")
-        action.replace("NULL", "null")
-        # .replace("_PyPegen_", "pegen.")
-        .replace("EXTRA", "...EXTRA")
-        .replace("CHECK ", "")  # CHECK is a version checker we could add this in later
-        # .replace("stmt_ty ,", "")
-        # .replace("expr_ty ,", "")
-        # .replace("( expor_ty )", "")
-        .replace(", p -> arena", "")
-        # .replace(" p ,", "")
-        .replace(", p ", "")
-        .replace(" -> ", ".")
-        # .replace("AugOperator *", "AugOperator")
-        .replace("RAISE", "pegen.RAISE")
-        .replace("NEW_TYPE_COMMENT", "pegen.NEW_TYPE_COMMENT")
-    )
-    # action = re.compile(r"^([A-Z]+[a-z]+[\w]*) ").sub(r" new astnodes.\1 ", action)  # astnode constructor
-    action = re.compile(r"\( ([bt]) \) \? \( \( expr_ty \) [bt] \)(.\w+) : null").sub(r"\1\2", action)
-    action = re.sub(r"pegen.augoperator \( (new astnodes.\w+) \)", r"\1", action)
-    action = re.sub(r"new astnodes.Py_([None|False|True|Ellipsis])", r"py\1", action)
+    # actions are separated by spaces and ?. is a single unit in javascript
+    action = action.replace("? .", "?.")
```

Comment:
`b ? b.id : null` vs `b?.id` reference.

Needed this line to account for javascript not liking `? .` 
(All actions get parsed with spaces between tokens)